### PR TITLE
Small ui improvents

### LINKED ISSUE #44: Multiple UI Improvements
- [x] back button for users
- [x] move save button
- [x] highlight selected field
- [x] warning when changing type of field
- [x] only a leave warning, when you actually change something in the slideeditview
- [x] better contrasts in light-mode

### LINKED ISSUE #46: Text-Editor enhancements
Suggestions for minor enhancements for the TextEditor:

- Currently, when opening the text-color options, the dropdowns get emptied because we dont have focus on the editor anymore
- Active options (bold, italics, underlined) aren't indicated that they are active

### LINKED ISSUE #52: SequenceEditView - LanguageSelector and LightDarkSwitch displayed in EmbedContent
When inputting an invalid url for embed content, in Firefox the LanguageSelector and the LightDarkSwitch are displayed on the slide.

### LINKED ISSUE #57: Reload Adminlist on AccountEdit
We need to reload the account list when an account is edited

### LINKED ISSUE #68: Only password repeat set is valid
When editing your own account data, setting only the repat password and leaving the normal on empty is accepted as a valid input

### LINKED ISSUE #72: Empty sequence name possible

### PR DESCRIPTION
**Pull request by Kr0nox**
All features from #44 are implemented (or at least checked of)

If anyone wants to add to this just do.
Otherwise ready for review

### CODE REVIEW

**Review by ProgramPhoenix — APPROVED** [review]

File: frontend/src/views/SequenceEditView.vue [hunk 1]
```diff
@@ -64,9 +72,11 @@
           :key="selectedSlideIndex"
           :slide="selectedSlide"
           :edit-mode="true"
+          :editing="currentEditingSlot ?? undefined"
           class="h-full absolute"
           @editing="
             (val) => {
+              addUnloadEventListener();
```
Comment (c0derMo) [inline frontend/src/views/SequenceEditView.vue]: Shouldn't stuff like changing the slide background color, title of sequence, adding slides, etc. also add the unload listener?

Comment (c0derMo) [reply frontend/src/views/SequenceEditView.vue]: @ProgramPhoenix

Comment (ProgramPhoenix) [reply frontend/src/views/SequenceEditView.vue]: review again should now work

File: frontend/src/components/sequenceEditOptionsTab/QuillToolBar.vue [hunk 2]
```diff
@@ -128,3 +126,14 @@ onMounted(() => {
     ?.toggleAttribute('selected', true);
 });
 </script>
+
+<style scoped>
+.active-indicator > .ql-active {
```
Comment (Kr0nox) [inline frontend/src/components/sequenceEditOptionsTab/QuillToolBar.vue:131]: change to `<style scoped lang="postcss"> and then use `@apply class` to apply tailwind classes for styling

File: playwright-report/index.html
Comment (Kr0nox) [inline playwright-report/index.html:1]: should this be pushed?

**Review by c0derMo — APPROVED** [review]

**Review by ProgramPhoenix — APPROVED** [review]

**Pull request merged**